src/components/Signup/Signup.jsx

```diff
@@ -1,129 +1,131 @@
+						<Form.Input label='Re-type Password'  type='Password' name='retype_password' value={retype_password} onChange={handleChange}  width='4' error={!validations.retype_password}/>
+					</Form.Group>
+					<Form.Group>
+						<Form.Input label='Email' placeholder='Email' width='4' name='email' value={email} onChange={handleChange} error={!validations.email} />
+						<Form.Field
+							compact
+							control={Select} 
+							options={months}
+							label='Month'
+							placeholder='Month'
+							name='month'
+							width='1'
+							value={month} 
+							onChange={handleChange}
+							error={!validations.month}
+						/>
+						<Form.Field
+							compact
+							control={Select} 
+							options={dates}
+							label='Date'
+							placeholder='Date'
+							width='1'
+							name='date'
+							value={date} 
+							onChange={handleChange}
+							error={!validations.date}
+						/>
+						<Form.Field
+							compact
+							control={Select}
+							options={years}
+							label='Year'
+							placeholder='Year'
+							width='2'
+							name='year'
+							value={year} 
+							onChange={handleChange}
+							error={!validations.year}
+						/>
+					</Form.Group>
+					<Form.Group >
+						<Form.Input label='City' placeholder='City' width='3' name='city' value={city} onChange={handleChange} error={!validations.city && !validations.city_zip_state}/>
+						<Form.Field
+							control={Select} 
+							options={states}
+							label='State'
+							placeholder='State'
+							width='3'
+							name='state'
+							value={state} 
+							onChange={handleChange}
+							error={!validations.state && !validations.city_zip_state}	
+						/>
+						<Form.Input label='Zipcode' placeholder='XXXXX' width='2' maxLength='5' name='zipcode' value={zipcode} onChange={handleChange} error={!validations.zipcode && !validations.city_zip_state} />
+					</Form.Group>
+					<Form.Field control={TextArea} label='About' placeholder='Tell us more about you...' maxLength='250' name='annotation' value={about} onChange={handleChange} />
+					<Form.Checkbox label='I agree to the Terms and Conditions' />
+					<div className="modal-ocean">
+						<strong>onChange:</strong>
+						<pre>{JSON.stringify({ first_name, email }, null, 2)}</pre>
+						<strong>onSubmit:</strong>
+						<pre>{JSON.stringify({ submittedName, submittedEmail }, null, 2)}</pre>
+						<input type="submit" id="submit-form" className="hidden" />
+					</div>
+				</Segment>
+			</Form>
+		</Modal.Content>
+		<Modal.Actions>
+			<label className="windows-btn-sign" onClick={upClose}><span className='window-btn-span-sign'>Sign In</span><Popup trigger={<Icon name="sign-in" />} content='Sign-in to your account' /></label>
+			<label className="windows-btn" htmlFor="submit-form" onClick={newClose} tabIndex="0"><span className='window-btn-span-ok'>Submit</span></label>
+			<label onClick={signBothClose} className="windows-btn"><span className='window-btn-span-cancel'>Cancel</span></label>
```

Comment:
Seems like this part is a little busy and these `<label>` tags have nested elements inside. Let's break them up in separate lines.